extensions/eclipse-che-theia-plugin-ext/src/node/che-plugin-service.ts

```diff
@@ -107,12 +107,9 @@ export class ChePluginServiceImpl implements ChePluginService {
         }
     }
 
-    private getAxiosInstance(): AxiosInstance {
-        if (!this.isItNode()) {
-            return axios;
-        }
-
+    private getAxiosInstance(uri: string): AxiosInstance {
```

Comment:
It's hard to understand what `uri` parameter means here, without examining the method internals.
Please, document it or consider replacing it with a flag, e.g. `attachCerts: boolean`.

Reply:
Done